cognite_toolkit/_api/run_api.py

```diff
@@ -19,11 +17,3 @@ def transformation(self, external_id: str) -> bool:
             self._create_tool_config(),
             external_id,
         )
-
-    def function(self, external_id: str, payload: dict, follow: bool = False) -> bool:
-        return RunFunctionCommand().run_function(
```

Comment:
Removing this is losing functionality. However, this is never used as far as I know (running the toolkit in a notebook). And the new function run is tied closely to which modules are available, so not sure how we should solve that here.